lte/gateway/c/core/oai/tasks/sgw_s8/sgw_s8_handlers.cpp

```diff
@@ -1846,17 +1850,30 @@ sgw_eps_bearer_context_information_t* update_sgw_context_to_s11_teid_map(
     OAILOG_FUNC_RETURN(LOG_SGW_S8, sgw_context_p);
   }
 
+  if (sgw_state->temporary_create_session_procedure_id_map.remove(
+          session_rsp_p->temporary_create_session_procedure_id, false) !=
```

Comment:
Why do we need the remove with `false` flag here? In the original logic the entry is being removed and freed from the hash table.

Reply:
In the original code, only entry is removed but not freed from hash table. This is taken care by API, hashtable_ts_remove() which only removes an entry from hash table but doesn't free. For API, hashtable_ts_free() free_wrapper is called as here https://github.com/magma/magma/blob/master/lte/gateway/c/core/oai/lib/hashtable/hashtable.c#L664
But same is not true for hashtable_ts_remove(), https://github.com/magma/magma/blob/master/lte/gateway/c/core/oai/lib/hashtable/hashtable.c#L691